telemetry/packages/e2e-tests/tests/test-todo.md

```diff
@@ -0,0 +1,105 @@
+# End to End Testing Briefing
+
+[Issue #102](https://github.com/Hyp-ed/hyped-2024/issues/102)
+
+(link branch for each testing topic in the list below once indv. branches have been pushed)
```

Comment:
kinda pointless to link each branch here if master will only have the merged ones, unless you wanna make a small PR to change this alongside each actual test PR

Reply:
yeah I agree, we'll be tracking it all in issue #102 anyway